plugin/conn_ip_example/conn_ip_example.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package test
```

Comment:
I will suggest you keep plugin unchanged. It is using `go build -buildmode=plugin` to build, and it is essentially a standalone package.

Reply:
Ah, ok.